include/engine/datafacade/datafacade_base.hpp

```diff
@@ -164,6 +164,9 @@ class BaseDataFacade
 
     virtual double GetMapMatchingMaxSpeed() const = 0;
 
+    virtual std::size_t GetMaxNumberOfAlternatives() const = 0;
```

Comment:
See above, should not be exposed in the datafacade. Adding this comment as a reminder to remove the interface.